packages/web-component/src/lib/descope-wc/DescopeWc.ts

```diff
@@ -322,8 +324,8 @@ class DescopeWc extends BaseDescopeWc {
     // If there is a start screen id, next action should start the flow
     // But if oidcIdpStateId is not empty, this optimization doesn't happen
     if (showFirstScreenOnExecutionInit(startScreenId, oidcIdpStateId)) {
-      stepStateUpdate.next = (interactionId, inputs) =>
-        this.sdk.flow.start(
+      stepStateUpdate.next = (interactionId, inputs) => {
+        return this.sdk.flow.start(
```

Comment:
In what case is it needed?

Reply:
if we want the invite to the flow to start with a screen - we will need it

Reply:
When flow starts with a screen for some reason
But we still want to take the token from the URL